common/beerocks/btl/CMakeLists.txt

```
@@ -12,9 +12,11 @@ find_package(MapfTransport QUIET)
 
 if(MapfTransport_FOUND)
 message(STATUS "${BoldGreen}Using MultiAP Framework for BTL transport${ColourReset}")
+set(BTL_TYPE "LOCAL_BUS" CACHE STRING "" FORCE)
```

This should be marked as INTERNAL as well. Note that INTERNAL implies FORCE.
```suggestion
set(BTL_TYPE "LOCAL_BUS" CACHE STRING "" INTERNAL)
```

Also, even for internal cache variables, it's good practice to provide a non-empty docstring.

---

I couldn't find out what the docstring is and what I should set it to, can you enlighten me?